libraries/AP_GPS/AP_GPS.h

```diff
@@ -521,6 +521,16 @@ class AP_GPS
     AP_Int8 _num_sats_arm_min;
     AP_Int8 _status_arm_min;
 
+    bool _degraded_gps[GPS_MAX_RECEIVERS]; // Any GPS that has previously failed is considered degraded
+
+    // Auto_Switch
+    enum GPS_AUTO_SWITCH {
+        GPS_AUTO_SWITCH_USEFIRST = 0, // Effectively USEFIRST
+        GPS_AUTO_SWITCH_USEBEST = 1,
+        GPS_AUTO_SWITCH_BLEND = 2,
+        GPS_AUTO_SWITCH_USESECOND = 3
```

Comment:
Is there a reason why `USESECOND` is not 1, next to `USEFIRST`?

Reply:
Yep. I left the constants the same as what Ardupilot had prior to my change.

Once I added the ENUM, this isn't as big an issue since I already broke compatibility with Ardupilot, but I think it might help prevent long term confusion by keeping the state numbers the same.